src/styles/common.less

```diff
@@ -49,3 +49,9 @@ video {
 .iconfont {
   font-size: 26px;
 }
+
+iframe {
+  border: none;
+  margin-top: 20px;
```

Comment:
I'm not sure if we should add these margins in, the author of the post might not have intended there to be margins around their iframe.

Reply:
@bonustrack what do you think?

Reply:
The margin create an extra space in feed:
![image](https://user-images.githubusercontent.com/16245250/32568290-5c27dcce-c4f0-11e7-81ca-df5f4284011b.png)
This PR is good for solve the issue about iframe border, for the margin we can do it in another PR, we should use a specific style class for it.

Reply:
@Kennybll Did you want to solve the issue for the margins, what @bonustrack talked about above in this PR or in another PR? If you want to do it in another PR, please close this one :)